src/main/java/com/endava/soa_5ed/task2/regex/StringGenerator.java

```diff
@@ -0,0 +1,54 @@
+package com.endava.soa_5ed.task2.regex;
+
+public class StringGenerator {
+    /* Date Time and Regex - Exercise 1 */
+    final static String baseStr = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890,./;'[]\\<>?:\"{}|`~!@#$%^&*()_+-=";
```

Comment:
Initialize properties inside constructors (default or/and parametrized).

Reply:
Done, thanks